Sources/PackageLoading/Target+PkgConfig.swift

```diff
@@ -209,6 +215,8 @@ extension SystemPackageProviderDescription {
                 }
             }
             return try packages.map({ try AbsolutePath(validating: brewPrefix).appending(components: "opt", $0, "lib", "pkgconfig") })
+        case .macPorts:
+            return [try AbsolutePath(validating: "/opt/local/lib/pkgconfig")]
```

Comment:
MacPorts can be installed places other than /opt/local, right? Should we have a similar `macPortsPrefixOverride` that gets threaded through, similar to the one for Homebrew?

Reply:
1. It appears that the brewPrefix is primarily used in the unit test context. Correct me if I'm wrong.
2. Regarding the Homebrew prefix, the prefix (Most because different default path for different arch: /opt/homebrew for Apple Silicon, /usr/local for macOS Intel) is handled by `brew --prefix`, and I haven’t found an equivalent straightforward command for MacPorts to get the prefix.

> Occasionally a MacPorts developer may wish to install more than one MacPorts instance on the same host. Only one copy of MacPorts may use the default prefix /opt/local, so for additional installations, the --prefix option can be used, as shown below.

Additionally, to use a custom prefix for MacPorts, the user must install it from source, which is a rare case for most SwiftPM users.

Reply:
I'm open to continue adding support for custom prefix MacPorts installation.

But I think we can land the basic support first and if there is downstream user actually needs this, we can add it later.

Reply:
Fair enough.